Sources/web3swift/Web3/Web3+HttpProvider.swift

```diff
@@ -11,6 +11,7 @@ import Core
 public class Web3HttpProvider: Web3Provider {
     public var url: URL
     public var network: Networks?
+    public var policies: Policies = .auto
```

Comment:
I think this is what you meant? @yaroslavyaroslav

Reply:
- Are we happy?
- Yeah, we happy!)